modules/gitops-release-notes-1-4-13.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-4-13_{context}"]
+= Release notes for {gitops-title} 1.4.13
+
+{gitops-title} 1.4.13 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
```

Comment:
According to the support matrix, it also support OCP 4.7, can you please recheck